### PR TITLE
Add YouTube Regret component

### LINKED ISSUE #3688: Create a "Youtube Regret" component
We need to implement a "Youtube Regret" streamfield component that allows for an image, a heading, and a richtext block for housing a few paragraphs of text.

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3688 

Desktop desgin : https://redpen.io/kz80aa1b9eedc210b4
Mobile design: https://redpen.io/wvce47e03aa936efbf

I'm getting content and structure in first. Styling will come later.

Test page: https://foundation-mofostaging-pr-3711.herokuapp.com/en/campaigns/youtube-regrets/
Edit page: https://foundation-mofostaging-pr-3711.herokuapp.com/cms/pages/51/edit

### CODE REVIEW

File: network-api/networkapi/wagtailpages/customblocks/youtube_regret_block.py [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+    image = ImageChooserBlock(
+        required=False
+    )
+
+    imageAltText = blocks.CharBlock(
+        required=False,
+        help_text='Image description (for screen readers).'
+    )
+
+    story = blocks.TextBlock(
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/customblocks/youtube_regret_block.py]: All regret stories we have are plain text.

File: network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/youtube_regret_block.html [hunk 2]
```diff
@@ -1,5 +1,8 @@
-{% load wagtailcore_tags %}
 
-<div class="youtube-regret">
-  <h1>{{ value.heading }}</h1>
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="youtube-regret mb-5">
+  {% image value.image original class="mb-3" alt=value.imageAltText %}
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/youtube_regret_block.html]: `original` is likely to be changed into something else in a later PR

File: network-api/networkapi/wagtailpages/customblocks/youtube_regret_block.py (hunk 1 above)
Comment (Pomax) [reply network-api/networkapi/wagtailpages/customblocks/youtube_regret_block.py]: ah, no need for richtext then. Nice

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html [hunk 3]
```diff
@@ -28,9 +28,9 @@ <h1>{{ page.specific.headline }}</h1>
   <hr>
 
   <div class="row">
-    <div class="col-12">
+    <div class="col-md-8 offset-md-2">
       {% for block in page.regret_stories %}
-      {% include_block block %}
+      {% include_block block with index=forloop.counter %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html]: hahaha, nice solution

File: source/sass/views/youtube-regrets.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+#view-youtube-regrets {
```
Comment (mmmavis) [inline source/sass/views/youtube-regrets.scss]: We are going to have lots of style overrides for this page so I decided to keep all the overrides here than creating a separate SCSS file each Wagtail component (e.g., "youtube regret" component).

File: network-api/networkapi/templates/pages/base.html [hunk 5]
```diff
@@ -30,6 +30,7 @@
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Nunito+Sans:400,300,700,300i">
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Zilla+Slab:300,400,600,700,300i">
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:400,700">
+    {% block extra_font %}{% endblock %}
```
Comment (Pomax) [inline network-api/networkapi/templates/pages/base.html]:
```suggestion
    {% block extended_head %}{% endblock %}
```

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html [hunk 6]
```diff
@@ -1,10 +1,13 @@
 {% extends "pages/base.html" %}
 {% load wagtailcore_tags wagtailimages_tags homepage_tags card_tags wagtailmetadata_tags multi_image_tags %}
 
-{% block body_id %}youtube-regret{% endblock %}
+{% block body_id %}youtube-regrets{% endblock %}
+{% block extra_font %}
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html]:
```suggestion
{% block extended_head %}
```

**Pull request merged**